src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-tiering-form/rgw-bucket-tiering-form.component.ts

```diff
@@ -93,31 +113,31 @@ export class RgwBucketTieringFormComponent extends CdForm implements OnInit {
     if (
       this.isValidPrefix(rule?.['Prefix']) ||
       this.isValidPrefix(rule?.['Filter']?.['Prefix']) ||
-      this.isValidArray(rule?.['Filter']?.['Tags']) ||
+      this.isValidArray(rule?.['Filter']?.['Tag']) ||
       this.isValidPrefix(rule?.['Filter']?.['And']?.['Prefix']) ||
-      this.isValidArray(rule?.['Filter']?.['And']?.['Tags'])
+      this.isValidArray(rule?.['Filter']?.['And']?.['Tag'])
     ) {
       return true;
     }
     return false;
   }
 
   isValidPrefix(value: string) {
-    return value !== undefined && value !== '';
+    return value !== undefined && value !== '' && value !== null;
```

Comment:
we should avoid undefined checks. Use `null` based defaults and check them.
But here its falsy value check so it can be `return !!value`